app/gui2/src/stores/graph/index.ts

```diff
@@ -251,11 +250,10 @@ export const useGraphStore = defineStore('graph', () => {
 
   function createNodes(
```

Comment:
I would move the entire logic of creating nodes to nodeCreation composable, which would just read computed `method` from graph store. Now the responsibility split between graph store and nodeCreation composable is unclear.